stdlib/pervasives.gr

```diff
@@ -32,8 +39,9 @@ import {
   (<<),
   (>>>),
   (>>),
-} from "runtime/numbers"
-import { toString, print, concat as (++) } from "runtime/string"
+}
+include "runtime/string"
+from String use { toString, print, concat as (++) }
```

Comment:
In relation to my comment above, `runtime/string` making `String` available and `string.gr` doing the same is weird and we should have a better pattern.

Reply:
I don't think it's weird.

Reply:
#1637